### PR TITLE
Added test case for identity split

### PR DESCRIPTION
**Pull request by ljaljushkin**
### Changes

Added test case for identity split and added meaningful names for test cases for better debugging

### Reason for changes

Address the comment https://github.com/openvinotoolkit/nncf/pull/1244#discussion_r955930013

### Tests

test_model_pruning_analysis.py

### CODE REVIEW

**Comment by ljaljushkin** [comment]
@sooahleex @daniil-lyakhov

File: tests/torch/pruning/helpers.py [hunk 1]
```diff
@@ -620,6 +620,26 @@ def forward(self, x):
...
+    #         (conv2)
+    def __init__(self):
+        super().__init__()
+        self.conv1 = create_conv(1, 4, 1, 1)
+        self.conv2 = create_conv(4, 8, 1, 1)
+
+    def forward(self, x):
+        x = self.conv1(x)
+        y1 = torch.chunk(x, chunks=1, dim=1)
+        y1 = self.conv2(y1[0])
```
Comment (sooahleex) [inline tests/torch/pruning/helpers.py:639]: Now I know what case you're talking about. There are other cases with dim=1, but there are no cases with more than one as shown in this. But does this kind of case actually exist? Then, is there a model that uses chunk/split? However, it looks good to add it for now.

Comment (daniil-lyakhov) [reply tests/torch/pruning/helpers.py:639]: We caught identity concat in one of OTE model, so identity chunk is possible :)

Comment (sooahleex) [reply tests/torch/pruning/helpers.py:639]: I see. Thank you for checking this part in detail.

**Review by sooahleex — APPROVED** [review]

**Pull request merged**